src/actionTypes/article.js

```diff
@@ -1,5 +1,27 @@
-// Action Types
-export const PUBLISH_ARTICLE = 'PUBLISH_ARTICLE';
-export const PUBLISH_ARTICLE_SUCCESS = 'PUBLISH_ARTICLE_SUCCESS';
-export const PUBLISH_ARTICLE_FAILURE = 'PUBLISH_ARTICLE_FAILURE';
-export const PUBLISH_ARTICLE_REQUEST = 'PUBLISH_ARTICLE_REQUEST';
+// Create Article
+export const PUBLISH_ARTICLE = "PUBLISH_ARTICLE"
+export const PUBLISH_ARTICLE_SUCCESS = "PUBLISH_ARTICLE_SUCCESS"
+export const PUBLISH_ARTICLE_FAILURE = "PUBLISH_ARTICLE_FAILURE"
+export const PUBLISH_ARTICLE_REQUEST = "PUBLISH_ARTICLE_REQUEST"
+
+// Update  Article
+export const UPDATE_ARTICLE = "UPDATE_ARTICLE"
+export const UPDATE_ARTICLE_SUCCESS = "PUBLISH_ARTICLE_SUCCESS"
+export const UPDATE_ARTICLE_FAILURE = "UPDATE_ARTICLE_FAILURE"
+export const UPDATE_ARTICLE_REQUEST = "UPDATE_ARTICLE_REQUEST"
+
+// Delete article
+export const DELETE_ARTICLE = "DELETE_ARTICLE"
+export const DELETE_ARTICLE_SUCCESS = "DELETE_ARTICLE_SUCCESS"
+export const DELETE_ARTICLE_FAILURE = "DELETE_ARTICLE_FAILURE"
+export const DELETE_ARTICLE_REQUEST = "DELETE_ARTICLE_REQUEST"
+
+// Fetch Article
+export const FETCH_ARTICLE = "FETCH_ARTICLE"
```

Comment:
Is there any reason we have multiple fetch single article action types?

Reply:
done